modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -1143,6 +1143,9 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 
 		IndexWriterHelperUtil.updateDocument(companyId, documents);
 		IndexWriterHelperUtil.updateDocument("test123", 123, null, null);
+
+		_indexWriterHelper.updateDocuments(companyId, documents, commitImmediately);
+		_indexWriterHelper.updateDocuments("test123", 123, null, null);
```

Comment:
Since it's the same case and it will be on the same `JSONObject`, shouldn't be separated by the empty line.